NewForce-at-Mountwest/bangazonapi-team-crocodoom
# payment test function added

## Proposed changes.
This is the build of the Payment Type Controller, and Payment Test Controller. This Payment controller allows a developer to GET all entities in the Payment Type table in Database, GET a single entity in the Payment Type table, POST an entity into the Payment Type table, PUT an existing entity in the Payment table, and then DELETE a single entity in the Payment table. This controller in conjunction with the Payment model satisfies the requirements for issue #3.



## Related Issues
Ticket #3
## Steps To Test
Rebuild your bangazonDatabase by running the updated Bangazon-api.sql in Sql Server Management Studio.

Start the program by cd'ing into the BangazonAPI and using the command dotnet run. Once the program is running, open up the Postman desktop app and run the following commands:

select GET then paste http://localhost:63540/api/paymenttype into the field and click send. The result should be an array of all the payment types in the database

select GET then paste http://localhost:63540/api/paymenttype/1 or any other number that showed up in the previous query as PaymentId and click send. The result should be only that object of the specified Order

select POST, then paste http://localhost:63540/api/paymenttype into the field, then click Body underneath the field, then select raw, and then paste this snippet or make one similar

then click send. The result should be the new payment you made.

select PUT then http://localhost:63540/api/paymenttype or any other PaymentId , then click Body underneath the field, then select raw, and then paste this snippet or make one similar
"name": "Edit Test Payment"
You should get nothing back from this. When you run the GET query the computer you specified in your PUT query should show the updated, edited information you gave it.

select DELETE then paste http://localhost:63540/api/paymenttype/1  or any other PaymentId then click send. You should get nothing back from this. When you run the GET query the computer you specified in your DELETE query should no longer exist.
## What kind of change does this PR introduce?
Please check one of the following:
- [X ] New feature
